tests/test_matrix_compat.py

```
@@ -0,0 +1,152 @@
+
+from mmrelay.matrix import compat
+
+
+@pytest.fixture(autouse=True)
+def reset_capability_cache():
+    compat.reset_matrix_capabilities_cache()
+    yield
+    compat.reset_matrix_capabilities_cache()
+
+
+def _patch_detection(monkeypatch, distributions, modules):
```

Ruff (0.15.12)

[warning] Missing return type annotation for private function `_patch_detection`

Add return type annotation: `None`

(ANN202)